Sources/CIColor+Extensions.swift

```diff
@@ -0,0 +1,61 @@
+import CoreImage
+
+
+public extension CIColor {
+
+    /// Initializes a `CIColor` object with the specified `white` component value in all three (RGB) channels.
+    /// - Parameters:
+    ///   - white: The unpremultiplied component value that should be use for all three color channels.
+    ///   - alpha: The alpha (opacity) component of the color.
+    ///   - colorSpace: The color space in which to create the new color. This color space must conform to the `CGColorSpaceModel.rgb` color space model.
+    convenience init?(white: CGFloat, alpha: CGFloat = 1.0, colorSpace: CGColorSpace? = nil) {
```

Comment:
Why is the parameter called `white`?

Reply:
Well, it's the amount of white in a gray color. I was actually transferring an API that is available for `UIColor` and `NSColor` to `CIColor`.